chart/README.md

```diff
@@ -541,14 +541,64 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
+cluster to automatically scale up/down based on the resource demand such as CPU and Memory utilization.
+
+Note: When setting auto-scaling parameters please make sure to have sufficient hardware resources available in the
+underlying nodes meet the max pod demands.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`, and set
+the `hpa.resources.memory.enabled` parameter to `true` to enable memory-based autoscaling.
+
+   ```
+   --set hpa.enabled=true
+   --set hpa.resources.memory.enabled=true
+   ```
+
+HPA requires metrics-server to be installed in the cluster. You can install it by
+running the following
+command: `kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml`
+Also, defined resources limits for all the containers in the cluster are required for HPA to be able to properly compute
+metrics.
+
+### Examples
+
+Some example commands are shown below.
+
+   ```
+   helm install --namespace staging mycluster --dependency-update
+    ...
+    --set hpa.enabled=true
+    --set hpa.resources.memory.enabled=true
+    --set iq_server.resources.requests.cpu="500m"
+    --set iq_server.resources.limits.cpu="1000m"
+    --set iq_server.resources.requests.memory="700M"
+    --set iq_server.resources.limits.memory="1024M"
+    --set forwarder.resources.requests.cpu="75m"
+    --set forwarder.resources.limits.cpu="200m"
+    --set forwarder.resources.requests.memory="100M"
+    --set forwarder.resources.limits.memory="200M"
+    --set aggregator.resources.requests.cpu="75m"
+    --set aggregator.resources.limits.cpu="200m"
+    --set aggregator.resources.requests.memory="100M"
+    --set aggregator.resources.limits.memory="200M" \
```

Comment:
Does this needs setting? can't we default to some sensible values based on `hpa.enabled=true` in some way?

Reply:
these are not needed - we need to update the iq-server-deployment template for the sidecar

Reply:
IIUC this is the shortest param list for HPA to work. Commit: f775c04d6426d2fdd4167b8d95950293b154981a
@agustin-mrtz  please let me know it that is correct.